### PR TITLE
VACMS 18486 - PAW title and test

### PR DESCRIPTION
**Pull request by eselkin**
## Are you removing, renaming or moving a folder in this PR?
- [x] No, I'm not changing any folders (skip to Summary and delete the rest of this section)
- [ ] Yes, I'm removing, renaming or moving a folder

## Summary

- Changes the title for all pages to `Learn how the PACT Act may affect you | Veterans Affairs`
- Changes customizeTitle to ignore provided arguments and solely return above title.
- Sitewide team

## Related issue(s)

- https://github.com/department-of-veterans-affairs/va.gov-cms/issues/18486

## Testing done

- Manual testing (ran through questionnaire), updated automated testing and ran tests

## Screenshots

<details>
<summary>New title</summary>
Screenshot 2024-07-22 at 4 32 55 PM
Screenshot 2024-07-22 at 4 31 43 PM
</details>

## What areas of the site does it impact?

Pact Act Wizard

## Acceptance criteria

- [x] All title tags in the PAW flow say Learn how the PACT Act may effect you | Veterans Affairs
- [ ] Review by @laflannery 

### Quality Assurance & Testing

- [x] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [x] Linting warnings have been addressed
- [x] Documentation has been updated ([link to documentation](#) \*if necessary)
- [x] Screenshot of the developed feature is added
- [ ] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [x] Browser console contains no warnings or errors.
- [x] Events are being sent to the appropriate logging solution
- [x] Feature/bug has a monitor built into Datadog or Grafana (if applicable)
- [ ] 
### Authentication

- [x] Did you login to a local build and verify all authenticated routes work as expected with a test user (unauthenticated)

## Requested Feedback

Review `/pact-act-eligibility` flow either locally or on RI

### CODE REVIEW

**Review by chriskim2311 — DISMISSED** [review]
LGTM!

**Review by laflannery — DISMISSED** [review]
Looks good!

File: src/applications/pact-act/tests/utilities/customize-title.unit.spec.js [hunk 1]
```diff
@@ -4,11 +4,13 @@ import { customizeTitle } from '../../utilities/customize-title';
...
     expect(customizeTitle('This is a test H1')).to.equal(
-      'This is a test H1 | PACT Act | Veterans Affairs',
+      'Learn how the PACT Act may effect you | Veterans Affairs',
     );
   });
 
   it('should return the correct title without an H1', () => {
-    expect(customizeTitle()).to.equal('PACT Act | Veterans Affairs');
+    expect(customizeTitle()).to.equal(
+      'Learn how the PACT Act may effect you | Veterans Affairs',
```
Comment (randimays) [inline src/applications/pact-act/tests/utilities/customize-title.unit.spec.js:13]: @laflannery Should this be "affect"?

Comment (eselkin) [reply src/applications/pact-act/tests/utilities/customize-title.unit.spec.js:13]: I believe you're right. I copy pastad from the ticket. Let's verify with @laflannery

File: src/applications/pact-act/utilities/customize-title.js [hunk 2]
```diff
@@ -1,9 +1,6 @@
 // Used to customize the <title> element in the head of the document
 // for accessibility
-export const customizeTitle = h1 => {
-  if (h1) {
-    return `${h1} | PACT Act | Veterans Affairs`;
-  }
-
-  return 'PACT Act | Veterans Affairs';
+export const customizeTitle = () => {
```
[the remark this answers is not in the document]
Comment (eselkin) [reply src/applications/pact-act/utilities/customize-title.js:3]: My reason for not doing that is what if in the future there's a desire to split the title between the flow and associated docs. I can definitely just code it without the function. What do you think about the possibility of future divisions of the title? @randimays

File: src/applications/pact-act/tests/utilities/customize-title.unit.spec.js (hunk 1 above)
Comment (laflannery) [reply src/applications/pact-act/tests/utilities/customize-title.unit.spec.js:13]: @randimays is absolutely correct. I **should** have copy pasted from the page lol

File: src/applications/pact-act/utilities/customize-title.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (randimays) [reply src/applications/pact-act/utilities/customize-title.js:3]: I think there's some risk in future-proofing as always. If we know a feature will go from simpler to more complex in the future, I think it's ok to make space for that. But if we're going from complex to simpler, I don't think it's necessary to keep the functionality in (particularly if it's a low lift -- we can always check Github and re-add old code).

[the remark this answers is not in the document]
Comment (eselkin) [reply src/applications/pact-act/utilities/customize-title.js:3]: Ok. Will remove

[the remark this answers is not in the document]
Comment (eselkin) [reply src/applications/pact-act/utilities/customize-title.js]: Removed function, uses, and test

File: src/applications/pact-act/tests/utilities/customize-title.unit.spec.js (hunk 1 above)
Comment (eselkin) [reply src/applications/pact-act/tests/utilities/customize-title.unit.spec.js]: Fixed, added screenshot with updated text

**Comment by eselkin** [comment]
Simplification (removal of customizeTitle function and hardcode string for title) - done
Removal of corresponding tests that I missed - done
Fixed spelling - done

**Review by laflannery — APPROVED** [review]

**Review by randimays — APPROVED** [review]
Nice, thanks @eselkin!

**Pull request merged**